app/views/coronavirus/announcements/edit.html.erb

```diff
@@ -1,15 +1,15 @@
 <%
 links = [
   {
-    text: 'Coronavirus pages',
+    text: t("coronavirus.pages.show.link_text"),
```

Comment:
This shouldn't refer to a key for a different view. It'd be more conventional to repeat the definition (as painful as that may seem).

Reply:
I'm not doing that. I'll leave it as hardcoded text.

Reply:
It might be conventional, but that's just pointless. It's a link back to the summary page, it should always be the same.

Reply:
A better solution is to move all of that links stuff to a shared partial. Something for another day.

Reply:
Yup, a partial sounds like the right approach to avoid the repetition.

I don't mind it as hardcoded text, that seems to have a much lower contagion risk than sharing the view copy.